.github/workflows/run-forge.yaml

```diff
@@ -32,6 +32,8 @@ env:
   FORGE_COMMENT: forge_comment.txt
   FORGE_PRE_COMMENT: forge_pre_comment.txt
   FORGE_RUNNER_MODE: k8s
+  FORGE_RUNNER_DURATION_SECS: 300
```

Comment:
you will also need to declare these in the `inputs` section above. This allows other workflows to call this workflow and specify these variables